app/app.rb

```diff
@@ -0,0 +1,86 @@
+ENV['RACK_ENV'] ||= 'development'
+
+require 'sinatra/base'
+require './app/data_mapper_setup'
+require 'bcrypt'
+require 'sinatra/flash'
+
+class Chitter < Sinatra::Base
+  register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+  use Rack::MethodOverride
+
+  get '/' do
+    redirect '/peeps'
+  end
+
+  get '/sessions/new' do
+    erb :'sessions/new'
+  end
+
+  post'/sessions' do
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.